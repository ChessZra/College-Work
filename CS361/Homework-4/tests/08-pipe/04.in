rm t10t10.txt 2>/dev/null;echo What a long test this is|sed s/long/interesting/g|sed s/test/course/g|sed s/this/CS361/g >t10t10.txt;cat -n<t10t10.txt;/usr/bin/wc t10t10.txt;rm t10t10.txt
