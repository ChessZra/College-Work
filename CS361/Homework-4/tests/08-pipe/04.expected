cs361sh>      1	What a interesting course CS361 is
 1  6 35 t10t10.txt
cs361sh> 
